### DISCUSSION TITLE
Diff between working tree and fetched remoted

### DISCUSSION THREAD

**Discussion by MrBrN197** [gitkraken/vscode-gitlens#2164] [Q&A] [answered]
I just installed Gitlens and I have a small confusion. 

## Scenario
I've pulled the from the remote branch.
the remote branch added some new files. 
![image](https://user-images.githubusercontent.com/18730106/186927174-04633872-a9e4-4fa8-8322-f4526dd724fa.png)

when I look at the gitlens sidebar on the commits section. I can see it claims I have 3 commits to pull from the remote branch
and there 2 files changed, 
![image](https://user-images.githubusercontent.com/18730106/186927686-c9a30601-5dc4-44ea-aba5-4ddafff51f59.png)

## Problem
however if I click on a file I don't see the changes on the diff view
![image](https://user-images.githubusercontent.com/18730106/186930017-806a2b1d-fa90-4545-8e02-557d8e6d19c3.png)

shouldn't this show me how the new file looks with all its content? I thought one diff would show what the working tree has and the other would show what the remote branch has 

Note: when opening the dropdown on the file and clicking each individual commit it shows correctly what I would expect.
![image](https://user-images.githubusercontent.com/18730106/186930077-4356cee2-f99e-44ee-a54c-7bb7c88565e2.png)

**Comment by eamodio** [comment, marked as answer]
This is a bug, I've opened https://github.com/gitkraken/vscode-gitlens/issues/2167 to track it.
